event_routing_backends/backends/events_router.py

```diff
@@ -39,6 +39,10 @@ def send(self, event):
         Arguments:
             event (dict):      original event dictionary
         """
+        if not routers:
```

Comment:
but the variable `routers` is defined for the first time on line 71, or am I missing something?